src/Esfa.Vacancy.Api.Types/GeoPoint.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Esfa.Vacancy.Api.Types
+{
+    public class GeoPoint
```

Comment:
This could be a struct 💥

Reply:
If I understand C# in this crazy time.

Reply:
This may mean it isn't nullable and you won't have to set it in the ctor of ApprenticeshipSummary

Reply:
Read this... https://stackoverflow.com/a/3729885/3917716